boards/esp32-ethernet-kit/Makefile.features

```diff
@@ -0,0 +1,10 @@
+CPU_MODEL = esp32-wrover
+
+# common board and CPU features
+include $(RIOTBOARD)/common/esp32/Makefile.features
+
+# additional features provided by the board
+FEATURES_PROVIDED += periph_adc
+FEATURES_PROVIDED += periph_i2c
+FEATURES_PROVIDED += periph_pwm
```

Comment:
Since you didn't configure any PWM GPIOs, you should probably drop this.

Reply:
Added PWM pins. I2C and SPI seem to be able to use any, so not sure if I should do something there.

Reply:
Hm, if there are GPIOs left that are broken out, you could define them as I2C and SPI pins. It would be also possible to define the same pins as PWM and as I2C pins. As long as modules `periph_pwm` and `periph_i2c` are used exclusively. You could define `FEATURES_CONFLICT += periph_pwm:periph_i2c` in that case. I have to analyze the schematic of the board a bit more. I will provide a suggestion later today.

But if you don't define I2C and SPI pins, you have to remove the features `periph_i2c` and `periph_spi`.